### PR TITLE
suggestion from #12

### PR DESCRIPTION
**Pull request by CoMPaTech**
See #12

### CODE REVIEW

**Comment by CoMPaTech** [comment]
While at it, clean fugly `.,` end of lines?

**Comment by stappersg** [comment]
What a poor commit message.

File: docs/lpic2.204.1.md [hunk 1]
```diff
@@ -229,24 +229,26 @@ listed below
...
+#### Build 
+Does not create array superblocks, and therefore does not destroy any pre 
+existing data. May be useful when attempting to recover or access stale 
+data. (can not be used in combination with `mdadm.conf `). Typically used 
+with legacy arrays, and rarely used.
 
 #### Create
 
 Creates an array from scratch, using
-pre-existing block devices, and activates the array.,
```
Comment (stappersg) [inline docs/lpic2.204.1.md:241]: Doesn't match with commit message.

File: docs/lpic2.204.1.md [hunk 2]
```diff
@@ -229,24 +229,26 @@ listed below
...
-pre-existing block devices, and activates the array., 
+pre-existing block devices, and activates the array.
 
 #### Grow
 
 
 Used to
 modify a existing array, for example adding, or removing devices.
 Capability is expected to be extended during the development lifecycle
-of the 2.6 kernel.,
```
Comment (stappersg) [inline docs/lpic2.204.1.md:249]: Doesn't match with commit message.

**Review by stappersg — CHANGES_REQUESTED** [review]
Please add **matching** commit message to commits.

**Comment by CoMPaTech** [comment]
Apparently you know how to PR yourself, awaiting your PR

**Pull request closed without merging**